src/components/block/InteractiveMap/MapKey.module.css

```diff
@@ -0,0 +1,22 @@
+.mapKey {
+  background-color: #fff;
+  bottom: 50px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  left: 60px;
+  padding: 30px 35px 30px 30px;
+  position: absolute;
+}
+
+.mapKeyEntry {
+  align-items: center;
+  display: flex;
+}
+
+.mapKeyEntry:not(:last-child) {
+  margin-bottom: 20px;
+}
```

Comment:
This is kind of new, but could we replace this entire pseudoclass selector with the [`row-gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap) property on the regular `.mapKeyEntry` selector? We're already using it in other places in the codebase, so if we do need to support older browsers, then we'll need to update every instance of it anyway.